src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="DatastoreStateEnum" type="enum8">
+    <cluster code="0x0752"/>
+    <item name="Pending" value="0x00"/>
+    <item name="Committed" value="0x01"/>
+    <item name="DeletePending" value="0x02"/>
+    <item name="CommitFailed" value="0x03"/>
+  </enum>
+
+  <struct name="DatastoreStatusEntryStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="State" type="DatastoreStateEnum" default="0x00" min="0x00" max="0x03"/>
+    <item fieldId="1" name="UpdateTimestamp" type="epoch_s"/>
+    <item fieldId="2" name="FailureCode" type="status" default="0"/>
+  </struct>
+
+  <struct name="DatastoreNodeKeySetEntry" apiMaturity="provisional" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="NodeID" type="node_id"/>
+    <item fieldId="1" name="GroupKeySetID" type="int16u"/>
+    <item fieldId="2" name="StatusEntry" type="DatastoreStatusEntryStruct"/>
+  </struct>
+
+  <enum name="DatastoreAccessControlEntryPrivilegeEnum" type="enum8">
+    <cluster code="0x0752"/>
+    <item name="View" value="0x01"/>
+    <item name="ProxyView" value="0x02"/>
+    <item name="Operate" value="0x03"/>
+    <item name="Manage" value="0x04"/>
+    <item name="Administer" value="0x05"/>
+  </enum>
+
+  <struct name="DatastoreGroupInformationEntryStruct" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="GroupID" type="int64u"/>
+    <item fieldId="1" name="FriendlyName" type="char_string" length="32"/>
+    <item fieldId="2" name="GroupKeySetID" type="int16u" isNullable="true" min="1" max="65534"/>
+    <item fieldId="3" name="GroupCAT" type="int16u" isNullable="true" min="1" max="65534"/>
+    <item fieldId="4" name="GroupCATVersion" type="int16u" isNullable="true" min="1" max="65534"/>
+    <item fieldId="5" name="GroupPermission" type="DatastoreAccessControlEntryPrivilegeEnum" min="0x00" max="0x05"/>
+  </struct>
+
+  <struct name="DatastoreBindingTargetStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="1" name="Node" type="node_id" optional="true"/>
+    <item fieldId="2" name="Group" type="group_id" optional="true" min="1"/>
+    <item fieldId="3" name="Endpoint" type="endpoint_no" optional="true"/>
+    <item fieldId="4" name="Cluster" type="cluster_id" optional="true"/>
+  </struct>
+
+  <struct name="DatastoreEndpointBindingEntryStruct" isFabricScoped="true">
```

Comment:
Spec issue: why is this a subsection under "11.24.5.7. DatastoreBindingTargetStruct Type"?   I would have expected this to be a sibling section.

Reply:
Tracked by https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620

Reply:
The spec change will be tracked using https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11622. Once the spec update is complete, an SDK PR can be submitted to align it with the updated specification.